.github/workflows/e2e_targeted.yaml

```diff
@@ -1,12 +1,14 @@
 name: e2e tests (targeted)
 
 on:
+  schedule:
+    - cron: '30 3 * * *'
```

Comment:
Is this change intended? We already have a cron job that runs e2es and it's configured in `e2e.yaml`.

Reply:
You actually made me realize that we might not necessarily need it.

This pipeline, comparing to e2e pipeline (not targeted 🤷 ) builds the image to be used. So it's handy for running against a random branch/tag to get KIC image from code of that branch/tag.

Since e2e run every day with `kong/nightly-ingress-controller:nightly` image, I agree we should be fine to leave that as the only job on schedule.

Having said all that I believe those 2 workflows could be merged at some point but I don't want to bother with it in this PR :)